test/templates/tooltip/component.njk

```diff
@@ -0,0 +1,19 @@
+<!--
+tooltip:
+    properties={},
+    modifier={ block: [] }
+-->
+{% macro tooltip(properties={}, modifier={}, state={}, aria={}) -%}
+<span class="tn-tooltip"{{ aria | aria }}>
+    <span class="tn-tooltip__content{{ modifier.arrow | modifier('tooltip__content') }}">
```

Comment:
The modifier key should match the element exactly. It's confusing to use `arrow` yet it modifies `content`.